src/lib.rs

```diff
@@ -77,7 +77,7 @@ impl Mul<f32> for Rgb {
             b: (f32::from(self.b) * rhs).max(0.0).min(255.0) as u8
         };
 
-        trace!("Scaling RGB by {} from {:?} to {:?}", rhs, self, result);
+        trace!("scaling RGB by {} from {:?} to {:?}", rhs, self, result);
```

Comment:
Are all `trace` messages uncapitalized, but other log messages are capitalized? What's the reasoning here? Everything above and equal to `info` is directly made for the user, while everything below are just random log messages?

Reply:
Yea I was imaginging a designation between standard user messages and developer messages. I'm happy to remove this distinction, but I personally kinda like having it.

Reply:
I think this would make it significantly harder for new contributors to follow our standard. Which means that you (or someone else) will likely have to do this whole refactoring again in a couple of months.

I think it would be beneficial to keep the format the same. So when I write a new `info!` message and there's another `trace!` message on screen, I can just use that message as reference for how I should write log messages in general, instead of having to find another info message.